src/main/java/at/hannibal2/skyhanni/features/garden/farming/FarmingWeightDisplay.kt

```diff
@@ -137,7 +137,7 @@ class FarmingWeightDisplay {
         }
 
         private fun update() {
-            if (!GardenAPI.inGarden()) return
+            if (!isEnabled()) return
```

Comment:
Please make it so that it only updates once when outside garden. Also you should handle world switches differently. no need to update the weight display on every world switch when the user wants to see the weight outside skyblock. also the user will ask for an optoin to show the display outside garden but not outside sb. maybe we should add that as well? personally i dont think this whole feature is needed for the farming weight display at all though.

Reply:
So should I return if the player is outside of sb in the onWorldChange event?
There's already an option to show it outside the garden but not sb. Also, people who pretty much only farm (like me), might always want to be able to see their weight. Even outside of sb

Reply:
I'm not sure how to do this. It's more complicated than just doing this:
> So I should return if the player is outside of sb and the weight is valid in the onTick event?
And return in the onWorldChange event if the player is outside of sb?

Reply:
Both are no longer necessary, made it so it will normally update outside SkyBlock as inside SkyBlock.